jobs/candlepinPipeline.groovy

```diff
@@ -144,7 +144,13 @@ stage('test') {
                 body: "${subject}:\n\nCheck console output at ${env.BUILD_URL} to view the results",
                 to: emailDestination,
             )
-            sendIrcNotification('#candlepin', "candlepin build # ${env.BUILD_NUMBER} failed. See ${env.BUILD_URL} for details.")
+            try {
+              branch = "${env.GIT_BRANCH}"
+              pr = branch.split('/')[2]
```

Comment:
assumes "origin/pr/1715/merge" , if the field is blank or in another format, this array index fails and the catch case is run.